templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  
+  <form action="{% url 'login' %}" method="POST">
+    {{ form.as_p }}
+    
+    {% csrf_token %}
+    <input type="submit" value="Submit">
+    <input type="hidden" name="next" value="{{ next }}">
```

Comment:
Ensure that the `next` parameter is properly validated and sanitized to prevent open redirect vulnerabilities. This is important for security when redirecting users after login.